packages/core/components/HeaderRibbon/HelpControl.tsx

```diff
@@ -0,0 +1,357 @@
+import {
```

Comment:
Would it be possible for us to separate out the actual tutorial text into a `Tutorials` folder, with individual files containing individual tutorial groupings? E.g.

```
Tutorials/
 - Filtering.tsx
 - Sorting.tsx
 - etc.tsx
```
I think it would be easier to work with this feature in the future of our code organization more closely matches the UI organization. Would also make this PR easier to review!

Reply:
Yeah good idea 👍